decidim-core/app/assets/stylesheets/decidim/utils/_helpers.scss

```diff
@@ -23,3 +23,17 @@
     margin-bottom: 0;
   }
 }
+
+.baseline-center{
```

Comment:
That's a workaround to align elements. I'll iterate over it in future approachs.